### PR TITLE
Improve error handling, fix #416

### LINKED ISSUE #416: Improve error handling
In some places like for example [here](https://github.com/kubernetes-incubator/kompose/blob/master/pkg/transformer/kubernetes/kubernetes.go#L491) we are printing errors using `logrus.Fatalf` deep inside code.  

It would be better if function returned error that can be processed outside of this function (higher in call stack). Code will be cleaner and it will be easier to write unit tests.

I think that we should invest some time in cleaning all error handling in Kompose codebase.

### PR DESCRIPTION
**Pull request by containscafeine**

### CODE REVIEW

**Comment by containscafeine** [comment]
Phew, got the tests passing.
Up for review, ping @cdrage @kadel

File: pkg/loader/bundle/bundle.go [hunk 1]
```diff
@@ -170,24 +171,24 @@ func loadPorts(service Service) ([]kobject.Ports, string) {
...
+func (b *Bundle) LoadFile(files []string) (kobject.KomposeObject, error) {
 	komposeObject := kobject.KomposeObject{
 		ServiceConfigs: make(map[string]kobject.ServiceConfig),
 		LoadedFrom:     "bundle",
 	}
 	file := files[0]
 	buf, err := ioutil.ReadFile(file)
 	if err != nil {
-		log.Fatalf("Failed to read bundles file: %s ", err)
+		return kobject.KomposeObject{}, errors.Wrap(err, "ioutil.ReadFile failed, Failed to read bundles file")
```
Comment (surajssd) [inline pkg/loader/bundle/bundle.go]: Just curious: Is it good practice to send the method/function name in error?

Comment (surajssd) [inline pkg/loader/bundle/bundle.go]: Also follow up question here you are using the error message as `ioutil.ReadFile` failed which is kinda absolute name but below at [line#191](https://github.com/kubernetes-incubator/kompose/pull/462/files#diff-fe05d91dde638f14ac7bf8afa193be77R191) you have just used the name which is `loadFile`, why is that so?

Comment (containscafeine) [inline pkg/loader/bundle/bundle.go]: @surajssd about the good practice part, it's a very subjective opinion, but we can use it to track down the error when the error messages will get printed in app.go

and about the the error message part, I added the name of the failing functions wrt the function they were failing in, so it gets easier to track down the error.

**Comment by cdrage** [comment]
Good job man! Did a review and the code LGTM :+1: 

Just need to rebase against master (I updated the vendoring in a different PR)

**Comment by containscafeine** [comment]
@cdrage fixed! :)

File: pkg/app/app.go [hunk 2]
```diff
@@ -35,6 +35,7 @@ import (
 
 	"os"
 
+	"github.com/fsouza/go-dockerclient/external/github.com/Sirupsen/logrus"
```
Comment (cdrage) [inline pkg/app/app.go]: why not import github.com/Sirupsen/logrus instead of from fsouza

Comment (containscafeine) [inline pkg/app/app.go]: Whoops, never realized that `logrus` was aliased to `log`, so my editor ended up importing this. Fixed! Nice catch, man :)

**Comment by containscafeine** [comment]
@cdrage fixed.

**Comment by cdrage** [comment]
LGTM! :+1:

**Pull request merged**